source/common/http/conn_pool_base.h

```diff
@@ -199,10 +198,9 @@ class MultiplexedActiveClientBase : public CodecClientCallbacks,
                                     public Http::ConnectionCallbacks,
                                     public Envoy::Http::ActiveClient {
 public:
-  MultiplexedActiveClientBase(HttpConnPoolImplBase& parent, uint32_t max_concurrent_streams,
-                              Stats::Counter& cx_total);
-  MultiplexedActiveClientBase(HttpConnPoolImplBase& parent, uint32_t max_concurrent_streams,
-                              Stats::Counter& cx_total, Upstream::Host::CreateConnectionData& data);
+  MultiplexedActiveClientBase(HttpConnPoolImplBase& parent, uint32_t effective_concurrent_streams,
+                              uint32_t max_configured_concurrent_streams, Stats::Counter& cx_total,
+                              OptRef<Upstream::Host::CreateConnectionData> data);
```

Comment:
Thanks for cleaning this up! It's annoying when there are N diff arguments to a constructor an N! various constructors.

Reply:
oh my gosh it was so hard adding my argument so I just gave up and changed all the things :-P